pow/src/main/java/tech/pegasys/artemis/pow/Eth1Manager.java

```diff
@@ -13,113 +13,121 @@
 
 package tech.pegasys.artemis.pow;
 
-import static tech.pegasys.artemis.util.config.Constants.ETH1_FOLLOW_DISTANCE;
+import static com.google.common.base.Preconditions.checkNotNull;
 
 import com.google.common.primitives.UnsignedLong;
-import io.reactivex.disposables.Disposable;
 import java.math.BigInteger;
-import java.util.concurrent.TimeUnit;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.apache.tuweni.bytes.Bytes32;
 import org.web3j.protocol.core.methods.response.EthBlock;
-import tech.pegasys.artemis.pow.api.MinGenesisTimeBlockEventChannel;
+import tech.pegasys.artemis.pow.api.Eth1EventsChannel;
 import tech.pegasys.artemis.pow.event.MinGenesisTimeBlockEvent;
 import tech.pegasys.artemis.util.async.AsyncRunner;
 import tech.pegasys.artemis.util.async.SafeFuture;
 import tech.pegasys.artemis.util.config.Constants;
 
-public class Eth1MinGenesisTimeBlockFinder {
+public class Eth1Manager {
 
   private static final Logger LOG = LogManager.getLogger();
 
   private final Eth1Provider eth1Provider;
-  private final MinGenesisTimeBlockEventChannel minGenesisTimeBlockEventChannel;
   private final AsyncRunner asyncRunner;
```

Comment:
Appears to be unused, but may be needed for some of the error handling that's currently missing.